scripts/RemoteTests.json

```diff
@@ -5,9 +5,9 @@
             "Platform": "Windows",
             "Tls": ["stub", "schannel", "mitls"],
             "Arch": ["x64", "x86", "arm", "arm64"],
-            "Exe": "quicping",
+            "Exe": "quicperf",
             "Arguments": {
-                "All": "-listen:* -port:4433 -peer_uni:1 -connections:10",
+                "All": "-TestName:Throughput -ServerMode:1 -listen:* -port:4433 -peer_uni:1 -connections:10",
```

Comment:
If you're passing the `listen` arg, there shouldn't be any reason you need a separate `ServerMode` arg.

Reply:
The way the code is written, the separate tests don't share args. So when I need to know to start the server or client, I don't know that listen is even going to be an argument.

Reply:
Ok.